nemo/collections/tts/data/text_to_speech_dataset_lhotse.py

```diff
@@ -311,6 +370,357 @@ def __getitem__(self, cuts):
 
         return batch_dict
 
-
     def collate_fn(self, batch: List[dict]):
         return batch
+
+
+class MagpieTTSMonologueLhotseDataset(torch.utils.data.Dataset):
```

Comment:
does this class replace the MagpieTTSLhotseDataset? Why do we now have two?

Reply:
@edresson, do you still need this class for your project?

Reply:
confirmed with @Edresson, we can remove them.